cfgov/unprocessed/css/organisms/video-player.less

```diff
@@ -148,6 +148,11 @@
         transform: translate( -50%, -50% );
     }
 
+    &:focus-visible {
+        outline: 1px dotted #0072ce !important;
+        outline-offset: 1px !important;
```

Comment:
Are the `!important`s necessary? I could be wrong but I removed them locally and it seems to work fine without them.

Reply:
You are correct, those were leftover artifacts from the panic of nothing wanting to work haha. Fixed!